transforms/language/readability/dpk_readability/runtime.py

```diff
@@ -0,0 +1,130 @@
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import sys
+from argparse import ArgumentParser, Namespace
+
+from data_processing.data_access import DataAccessFactory
+from data_processing.runtime.pure_python import PythonTransformLauncher
+from data_processing.runtime.pure_python.runtime_configuration import (
+    PythonTransformRuntimeConfiguration,
+)
+from data_processing.transform import TransformConfiguration
+from data_processing.utils import CLIArgumentProvider, ParamsUtils, get_logger, str2bool
+from dpk_readability.common import (
+    cli_prefix,
+    contents_column_name_cli_param,
+    contents_column_name_default,
+    curriculum_cli_param,
+    curriculum_default,
+    short_name,
+)
+from dpk_readability.transform import ReadabilityTransform
+
+
+logger = get_logger(__name__)
+
+
+class ReadabilityTransformConfiguration(TransformConfiguration):
+    """
+    Provides support for configuring and using the associated Transform class include
+    configuration with CLI args and combining of metadata.
+    """
+
+    def __init__(self):
+        super().__init__(
+            name=short_name,
+            transform_class=ReadabilityTransform,
+        )
+        self.daf = None
+
+    def add_input_params(self, parser: ArgumentParser) -> None:
+        """
+        Add Transform-specific arguments to the given  parser.
+        This will be included in a dictionary used to initialize the NOOPTransform.
+        By convention a common prefix should be used for all transform-specific CLI args
+        (e.g, noop_, pii_, etc.)
+        """
+        parser.add_argument(
+            f"--{contents_column_name_cli_param}",
+            type=str,
+            required=False,
+            default=contents_column_name_default,
+            help="contents column name for input parquet table to transform",
+        )
+        parser.add_argument(
+            f"--{curriculum_cli_param}",
+            type=lambda x: bool(str2bool(x)),
+            required=False,
+            default=curriculum_default,
+            help="curriculum parameter for transform; select True for curriculum learning",
+        )
+
+    def apply_input_params(self, args: Namespace) -> bool:
+        """
+        Validate and apply the arguments that have been parsed
+        :param args: user defined arguments.
+        :return: True, if validate pass or False otherwise
+        """
+        # Capture the args that are specific to this transform
+        captured = CLIArgumentProvider.capture_parameters(args, cli_prefix, False)
```

Comment:
Consider using the prefixed variable names throughout your code and change the False to True.
```suggestion
        captured = CLIArgumentProvider.capture_parameters(args, cli_prefix, True)
```

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/965/commits/65494e00943d47a236e168891c0e6210d58dd484